testbed/addons.ts

```diff
@@ -0,0 +1,19 @@
+import cdk = require('@aws-cdk/core');
+import * as eks from '@aws-cdk/aws-eks';
+import { Flux, Repository } from './addons/flux/construct';
+
+export interface AddonsProps extends cdk.StackProps {
+  cluster: eks.Cluster
+  repositories: Repository[];
+}
+
+export class Addons extends cdk.Construct {
+  constructor(scope: cdk.Construct, id: string, props: AddonsProps) {
+    super(scope, id);
+    // Add FluxV2
+    new Flux(this, 'Flux', {
+      cluster: props.cluster,
+      repositories: props.repositories,
+    });
+  }
+}
```

Comment:
Can you make sure your editor is inserting newlines at the bottom of files automatically? This helps with readability.

Reply:
my default formatter in vscode for ts apparently doesn't add it lol, i think in fact it removes it :|

Reply:
Any chance you can fix this setting?

Reply:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files